app/src/main/java/com/jerboa/ui/components/settings/account/AccountSettings.kt

```diff
@@ -255,97 +253,77 @@ fun SettingsForm(
+                state = defaultListingTypeState,
+                values = supportedListingTypes,
+                valueToText = {
+                    AnnotatedString(getLocalizedListingTypeName(ctx, it))
+                },
+                title = { Text(text = stringResource(R.string.account_settings_default_listing_type)) },
+                summary = {
+                    Text(getLocalizedListingTypeName(ctx, defaultListingTypeState.value))
+                },
+            )
 
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = showNsfw,
-            title = {
-                Text(text = stringResource(R.string.account_settings_show_nsfw))
-            },
-        )
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = showAvatars,
-            title = {
-                Text(text = stringResource(R.string.account_settings_show_avatars))
-            },
-        )
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = showReadPosts,
-            title = {
-                Text(text = stringResource(R.string.account_settings_show_read_posts))
-            },
-        )
+            ListPreference(
+                type = ListPreferenceType.DROPDOWN_MENU,
+                state = defaultSortTypeState,
+                values = supportedSortTypes,
+                valueToText = {
+                    AnnotatedString(ctx.getString(it.data.longForm))
+                },
+                title = { Text(text = stringResource(R.string.account_settings_default_sort_type)) },
+                summary = {
+                    Text(ctx.getString(defaultSortTypeState.value.data.longForm))
+                },
+            )
 
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = botAccount,
-            title = {
-                Text(text = stringResource(R.string.account_settings_bot_account))
-            },
-        )
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = showBotAccount,
-            title = {
-                Text(text = stringResource(R.string.account_settings_show_bot_accounts))
-            },
-        )
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            state = showScores,
-            title = {
-                Text(text = stringResource(R.string.account_settings_show_scores))
-            },
-        )
-        SettingsCheckbox(
-            modifier = Modifier.height(SETTINGS_MENU_LINK_HEIGHT),
-            enabled = email.isNotEmpty(),
-            state = sendNotificationsToEmail,
-            title = {
-                Text(text = stringResource(R.string.account_settings_send_notifications_to_email))
-            },
-        )
-        // TODO
```

Comment:
Make issue for this if you intend to remove this TODO

Reply:
Done, #1453